src/ProductConstructionService/ProductConstructionService.Api/appsettings.json

```diff
@@ -8,7 +8,8 @@
             "Microsoft.AspNetCore": "Warning",
             "Microsoft.AspNetCore.HttpLogging.HttpLoggingMiddleware": "Information",
             "Microsoft.EntityFrameworkCore.Database.Command": "Warning",
-            "Microsoft.IdentityModel.LoggingExtensions.IdentityLoggerAdapter": "Warning"
+            "Microsoft.IdentityModel.LoggingExtensions.IdentityLoggerAdapter": "Warning",
+            "Microsoft.DotNet.GitHub.Authentication": "Warning"
```

Comment:
This one removes the "Cached token found for installation" log line that has little value.